eng/pipelines/vmr-sync.yml

```
@@ -0,0 +1,38 @@
+trigger:
```

Please also add on top:
```suggestion
pr: none

trigger:
```